src/components/Activity/activity-statuses.ts

```diff
@@ -0,0 +1,6 @@
+export enum ActivityStatus {
```

Comment:
Just a nitpick in terms of naming conventions here.

Since we are already referencing a constant called `ActivityStatuses`, it would make sense to define the enum something like this:

```ts
export enum ActivityStatus {
  Confirmed = 'ACTIVITY_STATUS_CONFIRMED',
  Failed = 'ACTIVITY_STATUS_FAILED',
  Pending = 'ACTIVITY_STATUS_PENDING',
  TimedOut = 'ACTIVITY_STATUS_TIMED_OUT'
}
```

Reply:
Done ✅